WebService/v1/Models/DeviceModelApiModel/DeviceModelSimulation.cs

```diff
@@ -85,5 +85,27 @@ public void ValidateInputRequest(ILogger log)
                 }
             }
         }
+
+        public List<string> ValidationHelper()
+        {
+            List<string> errors = new List<string>();
+
+            try
+            {
+                IntervalHelper.ValidateInterval(this.Interval);
+            }
+            catch (InvalidIntervalException)
+            {
+                errors.Add(NO_INTERVAL);
+            }
+
+            // Allow user to have an empty list of scripts which may upload later
```

Comment:
If 'Allow user to have an empty list of scripts which may upload later' is correct then why are we adding it to the error list?

Reply:
We are checking if the `Scripts` property is `null` instead of  checking the `Count` of `Scripts` is `0`. E.g:  `if (this.Scripts == null || this.Scripts.Count == 0)`.